packages/components/src/Icon/AddLinkIcon.tsx

```diff
@@ -10,7 +10,7 @@ export const AddLinkIcon = (props: IconProps): JSX.Element => {
     <>
       <path
         fill="currentColor"
-        d="M9.125 14.167H5.833c-1.153 0-2.135-.406-2.948-1.219-.812-.813-1.218-1.795-1.218-2.948s.406-2.135 1.218-2.948c.813-.813 1.795-1.219 2.948-1.219h3.292v1.75H5.833c-.694 0-1.27.23-1.729.688-.458.458-.687 1.034-.687 1.729 0 .695.229 1.271.687 1.729.459.459 1.035.688 1.729.688h3.292zm-2.458-3.292v-1.75h6.666v1.75zm4.208 3.292v-1.75h3.292c.694 0 1.27-.23 1.729-.688.458-.458.687-1.034.687-1.729 0-.695-.229-1.271-.687-1.729-.459-.459-1.035-.688-1.729-.688h-3.292v-1.75h3.292c1.153 0 2.135.406 2.948 1.219.812.813 1.218 1.795 1.218 2.948s-.406 2.135-1.218 2.948c-.813.813-1.795 1.219-2.948 1.219z"
+        d="M9.125 14.167H5.833q-1.729 0-2.948-1.219T1.667 10t1.218-2.948q1.219-1.22 2.948-1.219h3.292v1.75H5.833q-1.04 0-1.729.688-.687.687-.687 1.729t.687 1.729q.688.688 1.729.688h3.292zm-2.458-3.292v-1.75h6.666v1.75zm4.208 3.292v-1.75h3.292q1.04 0 1.729-.688.687-.687.687-1.729t-.687-1.729q-.688-.688-1.729-.688h-3.292v-1.75h3.292q1.729 0 2.948 1.219T18.333 10t-1.218 2.948q-1.219 1.22-2.948 1.219z"
```

Comment:
I'd hope these mass changes are only related to you installing `ts-node`, otherwise if svgo decides it wants to change the code every so often for no good reason, it opens us up to missing things changing. Even if we have Chromatic, as shown in the latest snapshot, it's flagged for changes anyway due to you adding new icons to the list, so if any of the icons after it did change, it can easily be missed thinking it was just a layout shift.

Personally I wouldn't have committed these changes in this PR, but instead have it separately so we can ensure it's consistent (or if it's just some once off change that reverts the next time we run the script) and safe.

Reply:
I feel like its more likely its SVGo trying to optimize the SVG further rather than ts-node. But I take you're point about the risk 🤔 Might exclude it and do a follow up PR to update

Reply:
I dropped the old commit and re-added these :) should be a much small change